README.md

```diff
@@ -76,34 +88,35 @@ This app is designed to provide you with insights into your coding activities th
     NEXT_PUBLIC_MIXPANEL=NEXT_PUBLIC_MIXPANEL
     TOKEN_ENC_PUB_KEY=TOKEN_ENC_PUB_KEY
     TOKEN_ENC_PRI_KEY=TOKEN_ENC_PRI_KEY
-  ```
-
-<h3>Optional Keys and Behaviors</h3>
-
-You can append these extra keys to your env file to unlock more functioanlity offered by third party services.
-  
-  - AWS Credentials
-    - These are optional, if you dont have an AWS account, just remove these keys fron the env file.
-    - Without AWS creds, the generate cards are stored under `unwrapped-cards` directory under the project. This is generated by default, and stores the cards in nested directories of the format `{ROOT}/unwrapped/unwrapped-cards/[user_login]`
-    ```
-    AWS_ACCESS_KEY_ID=AWS_ACCESS_KEY_ID
-    AWS_SECRET_ACCESS_KEY=AWS_SECRET_ACCESS_KEY
-    AWS_REGION=AWS_REGION
-    UNWRAPPED_PERSISTENCE_BUCKET_NAME=UNWRAPPED_PERSISTENCE_BUCKET_NAME
     ```
+
+## Optional Keys and Behaviors
+Enhance functionality using optional keys in your .env file:
+- #### AWS Credentials (Optional)
+  - These are optional, if you dont have an AWS account, just remove these keys fron the env file.
+  - Without AWS creds, the generate cards are stored under `unwrapped-cards` directory under the project.
+    This is generated by default, and stores the cards in nested directories of the format `{ROOT}/unwrapped/unwrapped-cards/[user_login]`
     
-  - [Mixpanel](https://mixpanel.com/) Creds can be added to track usage metrics.
-    ```
-    NEXT_PUBLIC_MIXPANEL=NEXT_PUBLIC_MIXPANEL
-    ```
-  - [Sentry](https://sentry.io/) Creds are optional for Issue Tracking.
-    ```
-    SENTRY_AUTH_TOKEN=SENTRY_AUTH_TOKEN
-    ```
 
+     ```bash
+      AWS_ACCESS_KEY_ID=AWS_ACCESS_KEY_ID
+      AWS_SECRET_ACCESS_KEY=AWS_SECRET_ACCESS_KEY
+      AWS_REGION=AWS_REGION
+      UNWRAPPED_PERSISTENCE_BUCKET_NAME=UNWRAPPED_PERSISTENCE_BUCKET_NAME
+     ```
+- #### Third-Party Services
+  - #### [Mixpanel](https://mixpanel.com) for usage metrics:
+   ```bash
+      NEXT_PUBLIC_MIXPANEL=NEXT_PUBLIC_MIXPANEL
+     ```
+  - #### [Sentry](https://sentry.io/welcome/)  for issue tracking::
+   ```bash
+      NEXT_PUBLIC_MIXPANEL=NEXT_PUBLIC_MIXPANEL
+     ```
```

Comment:
I'd say undo changes in this entire section.

The nested lists aren't needed here. They add indentation that doesn't make it look that much better either.

Making the service names bolder is nice.

No need to specify AWS as optional. This whole section is already optional.